website_sale_stock_provisioning_date/models/stock_move.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from pytz import timezone
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    date_expected_date = fields.Date(
```

Comment:
Isn't better to call it `date_provisioning`?

Reply:
It was my initial call, but it doesn't fit when the move isn't an incoming one, I think...

Reply:
Well, this field name is very weird right now. It's true the other is not the finest one, but is not bad at all I think.

Reply:
ok then